README.md

```diff
@@ -1,32 +1,14 @@
 # multilogger
 
-Multilogger is a small utility based on [logrus](https://github.com/sirupsen/logrus) that writes logs simultaneously to the console and to a file. Each of those logging methods can be given a different logging level. It exposes the same API as a regular logrus logger.
+Multilogger is a small logging wrapper based on [logrus](https://github.com/sirupsen/logrus) that writes logs simultaneously to the console,
+files or any other hook. Each of those logging methods can be given a different logging level. It exposes the same API as a regular `logrus` logger.
 
-## Example
+See [multilogger godoc](https://godoc.org/github.com/coveooss/multilogger) for full documentation.
 
-```go
-import (
-    "github.com/coveooss/multilogger"
-    "github.com/sirupsen/logrus"
-)
+## multicolor
 
-log = multilogger.New(logrus.InfoLevel, logrus.DebugLevel, "my_log_file.out", "app_name")
-log.Info("test")
-log.Debug("test")
-log.Fatal("test")
+There is also a sub package used to handle colors and attributes. It is based on [color](https://github.com/fatih/color) which is an archived project
+but still useful. We based our implementation on [ghishadow](https://github.com/ghishadow/color) fork since it is maintained and have been migrated
```

Comment:
There is nothing new in all the commits in ghishadow. Versions of fatih color still work even if it's not in go mod. I like the repo with stars better than the repo with 18 useless commits and no stars. Am I missing a benefit of that repo being with go modules?